README.md

```diff
@@ -245,6 +245,10 @@ session.
 
 If you lose these details at any stage you can always reauthorise by redirecting the user back to the Xero OAuth gateway.
 
+#### Branding Themes API
```

Comment:
@CloCkWeRX Added this. Apologies for all the changes, my IDE removed all the whitespace.